schematron/v2.1.0/L000_Prelim_Analysis.csv

```diff
@@ -0,0 +1,21 @@
+phase title,phase see,pattern title,pattern see,rule title,rule context,assert test,assert description,assert severity,notes
+Preliminary Analysis,ASHRAE 211 5.2.3,Document Structure,,,/,count(auc:BuildingSync) = 1,,,
+,,,,,/auc:BuildingSync,count(auc:Facilities) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities,count(auc:Facility) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility,count(auc:Sites) = 1,,,
+,,,,,,count(auc:Reports) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites,count(auc:Site) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site,count(auc:Buildings) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings,count(auc:Building) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports,count(auc:Report) >= 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report,count(auc:Scenarios) = 1,,,
+,,,,,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Reports/auc:Report/auc:Scenarios,count(auc:Scenario) >= 1,,,
+,,Measured Scenario,ASHRAE 211 5.2.3.1 and 5.2.3.2,,//auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured]/auc:LinkedPremises/auc:Building/auc:LinkedBuildingID,//auc:Buildings/auc:Building[@ID = current()/@IDref],Scenario of Measured type must be linked to the Building,,
+,,,,,//auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured],auc:AllResourceTotals/auc:AllResourceTotal/auc:SiteEnergyUseIntensity,,,
+,,,,,,auc:AllResourceTotals/auc:AllResourceTotal/auc:EnergyCostIndex,,,
+,,Benchmark Scenario,ASHRAE 211 5.2.3.3,,//auc:Scenario[auc:ScenarioType/auc:Benchmark]/auc:LinkedPremises/auc:Building/auc:LinkedBuildingID,//auc:Buildings/auc:Building[@ID = current()/@IDref],Scenario of Benchmark type must be linked to the Building,,
+,,,,,//auc:Scenario/auc:ScenarioType/auc:Benchmark,count(auc:BenchmarkType/*) > 0,,,
+,,,,,,auc:BenchmarkTool,,,
+,,,,,,auc:BenchmarkYear,,,
+,,Resource Use,,,//auc:Scenario[auc:ScenarioType/auc:CurrentBuilding/auc:CalculationMethod/auc:Measured],count(auc:ResourceUses) = 1,,,

```

Comment:
I don't think actually for this L000 we need to specify any `auc:ResourceUse`s for L000.  5.2.3 does not appear to say anything about it (applies to next line as well)

Reply:
I had copied that from the original L000 schematron. I agree though, I don't think it's necessary